package.json

```diff
@@ -29,16 +35,16 @@
   },
   "homepage": "https://github.com/TrySound/postcss-inline-svg",
   "dependencies": {
-    "css-select": "^2.0.0",
-    "dom-serializer": "^0.1.0",
-    "htmlparser2": "^3.9.0",
-    "postcss": "^6.0.21",
-    "postcss-value-parser": "^3.2.3"
+    "css-select": "^2.0.2",
+    "dom-serializer": "^0.1.1",
+    "htmlparser2": "^3.10.1",
+    "postcss": "^7.0.17",
```

Comment:
Is there a practice to have postcss in peer dependencies like with react?

Reply:
I'm not aware of such practice for PostCSS.